docs/configuration/tls.md

```diff
@@ -54,7 +62,7 @@ For basic TLS, only the fields `tls.crt` and `tls.key` are needed in the kube se
           selfSigned: {}
         EOF
 
-3.  Create a `Certificate` CR
+3.  Create a `Certificate` CR:
```

Comment:
since this is a executable scriptlet, we should define the `${HOSTNAME}` variable as part of the script block (or earlier in this guide) with reasonable default and instruct user to change the value like in the explanation below